### ISSUE TITLE
Pretrained network for fcn8s model

### ISSUE THREAD

**Issue by agarwal-shubham** [wkentaro/pytorch-fcn#60]
Hi, I tried to train the fcn8s model using pretrained weights from the vgg16 model, but there was no visible learning and the model output was all zero for the first to ~10k iterations. However, the training loss converges when I use the fcn16s model to initialize the weights. I could not really understand the reason behind this, can you help with an explanation.

**Comment by wkentaro** [comment]
Maybe you should read the original paper: https://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Long_Fully_Convolutional_Networks_2015_CVPR_paper.pdf

> Training from
scratch is not feasible considering the time required to
learn the base classification nets. (Note that the VGG net is
trained in stages, while we initialize from the full 16-layer

**Comment by agarwal-shubham** [comment]
What I wasn't able to understand is that why was it so much more worse to directly train the FCN16s using VGG16 (and not from scratch) (I understand that it makes sense to finetune FCN16s model from FCN32s). But as you said I revisited the paper and I think the proper reason is stated under **Refinement by other means** right before Section 4.3 in [the paper](https://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Long_Fully_Convolutional_Networks_2015_CVPR_paper.pdf)
Thanks :+1:
